cloud/docker_client.go

```diff
@@ -362,6 +370,30 @@ func (c *dockerClientImpl) CreateContainer(ctx context.Context, parentHost, cont
 	return nil
 }
 
+func (c *dockerClientImpl) GetLogs(ctx context.Context, h *host.Host, containerID string, options types.ContainerLogsOptions) (*LogReader, error) {
+	dockerClient, err := c.generateClient(h)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to generate docker client")
+	}
+
+	stream, err := dockerClient.ContainerLogs(ctx, containerID, options)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Docker logs API call failed for container %s", containerID)
+	}
+	res := LogReader{}
+	tempout := new(bytes.Buffer)
```

Comment:
nit I'd just do &bytes.Buffer{}

new is fine (and equiv) but I'd generally avoid it.